packages/app/components/Queue/TAQueueList.tsx

```diff
@@ -270,8 +298,20 @@ export default function TAQueueList({
           <Row justify="space-between">
             <QueueTitle>{queue?.room}</QueueTitle>
             <Row>
-              <HelpNextButton size="large">Help Next</HelpNextButton>
-              {queue?.staffList.some((e) => e.id === user.id) ? (
+              <Tooltip
+                title={
+                  !isStaffCheckedIn && "You must check in to help students!"
+                }
+              >
+                <HelpNextButton
+                  disabled={!isStaffCheckedIn}
+                  size="large"
+                  tooltip="test"
```

Comment:
Is this needed?

Reply:
prob not... will get to after work and then merge